app/controllers/assetlinks_controller.rb

```diff
@@ -0,0 +1,9 @@
+class AssetlinksController < ApplicationController
+  def apple_app_site_association
+    render :json => JSON.load(Base64.decode64 ENV.fetch('APPLE_APP_SITE_ASSOCIATION', 'e30='))
+  end
+
+  def android_assetlinks
+    render :json => JSON.load(Base64.decode64 ENV.fetch('ANDROID_ASSETLINKS', 'e30='))
```

Comment:
Toto mi pride zvlastne, nedame tu normalne jbuilder a to co tam ma byt ci je to tak velmi custom, ze to chceme cele davat do envu?